rules/xcodeproj.bzl

```diff
@@ -911,6 +938,8 @@ https://www.rubydoc.info/github/CocoaPods/Xcodeproj/Xcodeproj/Constants
         "additional_files": attr.label_list(allow_files = True, allow_empty = True, default = [], mandatory = False),
         "additional_prebuild_script": attr.string(default = "", mandatory = False),  # Note this script will run BEFORE Bazel build script
         "additional_bazel_build_options": attr.string_list(default = [], mandatory = False),
+        "additional_pre_actions": attr.string_list_dict(default = {}, mandatory = False, doc = "Configure a list of pre-actions for build/run etc. For each entry the key is one of build/test/run and value is a list of scripts"),
+        "additional_post_actions": attr.string_list_dict(default = {}, mandatory = False, doc = "Configure a list of post-actions, see additional_pre_actions for details"),
```

Comment:
Nit: I think it's fine to duplicate some of the doc here instead of pointing folks to the pre-actions's 

But up to you!